.github/workflows/stale.yml

```diff
@@ -0,0 +1,44 @@
+name: Close Stale Issues
+on:
+  schedule:
+    # Run at midnight every night
+    - cron: '0 0 * * *'
+permissions:
+  contents: write # only for delete-branch option
+  issues: write
+  pull-requests: write
```

Comment:
Why does it need PR permissions? The previous version only touched issues, if we don't need it let's not have it.

Reply:
you're right!